build.gradle

```diff
@@ -1,40 +1,14 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
-
-buildscript {
-    repositories {
-		google()
-		jcenter()
-    }
-	dependencies {
-		// NOTE - this is referencing a local variable that you must set in your gradle.properties
-		// in the top level folder.
-		//
-		// Example:
-		//
-		// project.ext {
-		//    localBuildTools = com.android.tools.build:gradle:0.14.1
-		// }
-		//
-		if (project.hasProperty('localBuildTools')) {
-			classpath project.ext.localBuildTools
-		} else {
-			// Fallback to newest build tools if property not set
-			classpath 'com.android.tools.build:gradle:7.1.1'
-		}
-	}
-}
-
 allprojects {
     repositories {
-		google()
-		jcenter()
+	google()
+	mavenCentral()
     }
 }
 
 ext {
-	compileSdkVersion = 28
-	minSdkVersion = 14
-	targetSdkVersion = 28
-	supportAnnotation = 'androidx.annotation:annotation:1.0.0'
-	facebookRebound = "com.facebook.rebound:rebound:0.3.8"
+    compileSdkVersion = 31
+    minSdkVersion = 14
+    targetSdkVersion = 31
+    supportAnnotation = 'androidx.annotation:annotation:1.0.0'
+    facebookRebound = "com.facebook.rebound:rebound:0.3.8"
```

Comment:
This seems a weird place to have the lib versions. Not for this PR, but we should use Gradle catalogs or similar https://docs.gradle.org/current/userguide/platforms.html#sub:conventional-dependencies-toml

i can work on this in future PR, and maybe we should extend that approach to other libraries

Reply:
We should certainly separate it in its own PR. I am a big fan of the new catalogs and it's actually my next project where we'll centralize all our project dependencies. I don't know if it'd make sense to combine Tumblr into that ecosystem, but I wouldn't mind working on a separate one for Tumblr client/libraries. Although, Tumblr client does already have a much nicer way of handling this, so I don't know if it'd be impactful enough.